src/luaot.c

```diff
@@ -766,7 +775,21 @@ void print_functions(Proto *p)
 }
 
 static
-void print_source_code()
+int bytecode_writer(lua_State *L, const void *p, size_t sz, void *ud)
+{
+    (void)L;
+    luaL_Buffer *b = (luaL_Buffer *)ud;
```

Comment:
In C, we don't need to write an explicit cast for a void*. I think it would suffice to say

    luaL_Buffer *b = ud;

Also, I think we can also add a line for

     const char *bytes = p
     
(I expect that we don't need to say "unsigned")

Reply:
>In C, we don't need to write an explicit cast for a void*. I think it would suffice to say
yeah that was a holdover when I had my own struct, accident! Thanks.
>Also, I think we can also add a line for
Will do